ddtrace/contrib/django/patch.py

```diff
@@ -44,6 +44,7 @@
         analytics_sample_rate=None,
         trace_query_string=None,  # Default to global config
         include_user_name=True,
+        use_legacy_resource_format=get_env("django", "use_legacy_resource_format", default=False),
```

Comment:
I wonder if we want to just be direct with this naming: `use_handler_resource_format`?

Reply:
👍